setup.py

```diff
@@ -72,7 +72,7 @@
     deps['p2p'] +
     deps['trinity'] +
     deps['test'] +
-    deps['doc']
```

Comment:
I don't totally disagree that `docs` is more natural than `doc`, but the other extras are all singular (like `test`). More importantly, all the other python ethereum projects that follow this extras pattern use `doc`. Inconsistency between these projects adds unnecessary friction. If you think it's important to change the name to `docs`, then let's go have a cleanup party on all the other repos to change them. If it's not important to you, can we go back to `doc`?

Reply:
+1 for consistency, don't care which one we use.

Reply:
True. Reasoning was that `docs` feels more natural and seems to be used more widespred (e.g. [ReadTheDocs](https://docs.readthedocs.io/en/latest/yaml-config.html)) and when we had the deps in an `txt` file it also was `requirements-docs.txt` as it is still for e.g [eth-account(]https://github.com/ethereum/eth-account/blob/master/requirements-docs.txt)

But I agree that is inconsistent in regard to the points you mention. Let me switch it back.